models.js

```diff
@@ -29,3 +29,11 @@ exports.selectArticles = () => {
     return resultArray[0].rows;
   });
 };
+
+exports.selectArticleById = (req) => {
+  const article_id = req.params.article_id;
+  return db.query("SELECT * FROM articles WHERE article_id = $1", [article_id])
+    .then(article => {
+    return article.rows
+  })
```

Comment:
We need some error handling here to check if there is not an article with that ID. Just move what you've done in the controller here 👍